cmd/scollector/collectors/redis_unix.go

```diff
@@ -239,10 +239,53 @@ func c_redis() (opentsdb.MultiDataPoint, error) {
 			Add(&md, "redis."+sp[0], sp[1], tags, metadata.Unknown, metadata.None, "")
 		}
 		Add(&md, "redis.key_count", keys, tags, metadata.Gauge, metadata.Key, descRedisKeyCount)
+
+		info_command, info_command_err := c.Do("info", "commandstats")
```

Comment:
Since the default info gets almost all the information anyways, I think we should only call info once with the all argument, and parse that.